rllib/algorithms/qmix/qmix.py

```diff
@@ -77,7 +77,10 @@ def __init__(self):
         self.double_q = True
         self.optim_alpha = 0.99
         self.optim_eps = 0.00001
-        self.grad_clip = 10
+
+        # TODO (sven): Deprecate grad_clip setting once all-in on new Learner API.
+        self.grad_clip = 10.0
+        self.grad_clip_by_global_norm = 10.0
```

Comment:
how is this going to be used downstream by the qmix policy / multi_gpu_train_one_batch.

Reply:
fixed by replacing the new settings with backward-compatible ones